hack/validate_yaml.py

```diff
@@ -0,0 +1,42 @@
+# Usage
```

Comment:
Where is this file used?

Reply:
IDK, ran `cp -R aws-account-operator/hack/. route-monitor-operator/hack` and kept the results, if it's not used I see no reason to keep it

Reply:
I would not keep it if it's not in use